Session.js

```diff
@@ -1679,9 +1681,11 @@ Session.prototype = {
 		var self = this;
 
 		return new Promise(function (resolve) {
-			var zip = new AdmZip();
-			zip.addLocalFile(filename);
-			var data = zip.toBuffer().toString('base64');
+			var content = fs.readFileSync(filename);
+
+			var zip = new JSZip();
+			zip.file(path.basename(filename), content);
+			var data = zip.generate({type: 'base64'});
 			zip = null;
```

Comment:
Keeping it from the old codebase. As I understand it helps with GC and avoiding memory leaks.